containers/ddev-webserver/Dockerfile

```diff
@@ -262,7 +261,7 @@ ADD ddev-webserver-prod-files /
 RUN phpdismod blackfire
 RUN phpdismod xhprof
 
-RUN curl --fail -sSL https://github.com/backdrop-contrib/drush/releases/download/${BACKDROP_DRUSH_VERSION}/backdrop-drush-extension.zip -o /tmp/backdrop-drush-extension.zip && unzip -o /tmp/backdrop-drush-extension.zip -d /var/tmp/backdrop_drush_commands && chmod -R ugo+w /var/tmp/backdrop_drush_commands && rm /tmp/backdrop-drush-extension.zip
+RUN set -o pipefail && tag=$(curl -L --fail --silent "https://api.github.com/repos/backdrop-contrib/drush/releases/latest" | jq -r .tag_name) && curl --fail -sSL "https://github.com/backdrop-contrib/drush/releases/download/${tag}/backdrop-drush-extension.zip" -o /tmp/backdrop-drush-extension.zip && unzip -o /tmp/backdrop-drush-extension.zip -d /var/tmp/backdrop_drush_commands && chmod -R ugo+w /var/tmp/backdrop_drush_commands && rm /tmp/backdrop-drush-extension.zip
```

Comment:
I approve of the backdrop update to get latest, but interested in what got you interested in it. (As you know from the issues, backdrop-drush seems to have dropped off in popularity.)

Reply:
I just checked the Dockerfile for hardcoded links like this and noticed the new release for backdrop-drush-extension.